providers/mondoo/resources/mondoo_organization.go

```diff
@@ -5,12 +5,29 @@ package resources
 
 import (
 	"context"
+	"errors"
 
 	"go.mondoo.com/cnquery/v11/llx"
+	"go.mondoo.com/cnquery/v11/providers-sdk/v1/plugin"
 	"go.mondoo.com/cnquery/v11/providers/mondoo/connection"
 	mondoogql "go.mondoo.com/mondoo-go"
 )
 
+func initMondooOrganization(runtime *plugin.Runtime, args map[string]*llx.RawData) (map[string]*llx.RawData, plugin.Resource, error) {
+	conn := runtime.Connection.(*connection.Connection)
+	if conn.Type != connection.ConnTypeOrganization {
+		return nil, nil, errors.New("cannot initialize mondoo.organization, invalid connection")
+	}
+	args["mrn"] = llx.StringData(conn.Upstream.SpaceMrn)
+	args["name"] = llx.StringData(connection.MrnBasenameOrMrn(conn.Upstream.SpaceMrn))
```

Comment:
That is the space id, right?

Reply:
this is either the org id or the space id, regardless of the fact we call it `SpaceMrn`, it all depends on what the mondoo config that we're connecting with has